lib/rubocop/cop/style/mixin_usage.rb

```diff
@@ -3,11 +3,8 @@
 module RuboCop
   module Cop
     module Style
-      # This cop checks that `include`, `extend` and `prepend` exists at
-      # the top level.
-      # Using these at the top level affects the behavior of `Object`.
-      # There will not be using `include`, `extend` and `prepend` at
-      # the top level. Let's use it inside `class` or `module`.
+      # This cop checks that `include`, `extend` and `prepend` statements appear
+      # inside classes and modules, not at the top level.
```

Comment:
Hm. This change seems to make the users unknown about "why" this style is preferred 🤔

Reply:
That's true of most descriptions, isn't it? IMO, the comment should indicate what the cop does. The style guide should indicate why that style is preferred.

Reply:
I don't think this is covered by an entry in the style guide, though.

Reply:
Fixed to include a description about why that style is preferred.